pkgs/development/python-modules/gpt4all/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, requests
+, tqdm
+, cmake
+, pytestCheckHook
+}:
+
+let
+  version = "1.0.8";
+  src = fetchFromGitHub {
+      owner = "nomic-ai";
+      repo = "gpt4all";
+      rev = "39acbc837816ea9c7673bfe5cf83aeaed95b2c5f";
```

Comment:
What about the tags?

Reply:
> Note that it's not the latest version using tags since it needs vulkan. I'm thinking we can merge this first and then I'll do another PR for the update.

https://github.com/NixOS/nixpkgs/pull/249553#issuecomment-1742178834

Reply:
Please add a comment which version this represents, why this isn't a tag and how that'll change once Vulkan support is figured out.